velox/exec/Window.cpp

```diff
@@ -368,7 +368,10 @@ void Window::callApplyForPartitionRows(
               isStartBound ? rawPeerStarts : rawPeerEnds;
           std::copy(rawPeerBuffer, rawPeerBuffer + numRows, frameBoundBuffer);
         } else {
-          VELOX_NYI("Not supported");
+          std::iota(
```

Comment:
Please add a comment describing the numbers that are getting populated in this buffer.

Reply:
Thanks for the suggestion, updated!